database/api/routes.md

```diff
@@ -30,26 +30,27 @@ NOTE:: recon json key has been removed from previous versions
             lastModified: String
         }
     ]
+    NOTE: Will these be sorted by program_block_id? Can we gurantee that a inProgramBlock has been created first?
     programBlock: [
         {
-            id: Integer
+            programBlockId: Integer
```

Comment:
let's please not change these any more unless its been discussed, it's a pain having to go back and redo the client code every time this changes.

Reply:
I changed it because it would have a weird conflict with the default Django id, and this was the route I ended up going with

Reply:
I could probably change these to id and use the serializers to specify the model's integer field if this change is a lot of work

Reply:
> I could probably change these to id and use the serializers to specify the model's integer field if this change is a lot of work

yes, that is possible.

I haven't written tests yet so it's not a big task to rework it again, but in the future, no api changes unless there is a very good reason and it is well communicated